deploy/chart/templates/0000_50_olm_10-operatorgroup.crd.yaml

```diff
@@ -4,11 +4,14 @@ metadata:
   name: operatorgroups.operators.coreos.com
 spec:
   group: operators.coreos.com
-  version: v1alpha2
+  version: v1
   versions:
-  - name: v1alpha2
-    served: true
-    storage: true
+    - name: v1
+      served: true
+      storage: true
+    - name: v1alpha2
```

Comment:
At some point we're going to need to do [something like this](https://v1-12.docs.kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version) to completely migrate storage versions - or we could wait for 1.14 to deprecate since there are CRD versioning behavioral differences.